README.md

```diff
@@ -11,14 +11,6 @@ classes and functions used in other Ignition libraries and programs.
 
 ## Continuous integration
 
-This project uses [appveyor](https://ci.appveyor.com/project/scpeters/ign-math/history)
```

Comment:
why did you remove this comment? appveyor is still configured for windows, right?

Reply:
I've been removing Appveyor and Drone.io because I assumed we'd move to GitHub pipelines. But we should probably get @scpeters 's ok on this. In any case, the current links are "invalid" until we re-setup Appveyor / Drone for GitHub.